sources/hr/countrywide.json

```diff
@@ -0,0 +1,34 @@
+{
+    "coverage": {
+        "ISO 3166": {
+            "alpha2": "HR",
+            "country": "Croatia"
+        },
+        "country": "hr"
+    },
+    "schema": 2,
+    "layers": {
+        "addresses": [
+            {
+                "name": "country",
+                "website": "https://inspire-geoportal.ec.europa.eu/srv/eng/catalog.search#/datasetdetails?country=hr&view=themeOverview&theme=none&resourceId=d3104048-688a-4305-9471-77288cc6e350",
+                "data": "https://geoportal.dgu.hr/services/atom/INSPIRE_Addresses_(AD).zip",
+                "frequency": "weekly",
+                "protocol": "http",
+                "compression": "zip",
+                "license": {
+                    "text": "no limitations to public access",
+                    "url": "https://inspire.ec.europa.eu/metadata-codelist/LimitationsOnPublicAccess/noLimitations"
+                },
+                "conform": {
+                    "encoding": "UTF-8",
+                    "format": "xml",
+                    "file": "Address.gml",
+                    "accuracy": 1,
+                    "number": "ad:designator",
+                    "street": "ad:thoroughfareName"
```

Comment:
Ran out of ideas how to map these values from GML XML file.... 
Pointers to any known working examples or documentation welcome.